src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -241,13 +255,21 @@ class MockClientBackend : public ClientBackend {
     return Error::Success;
   }
 
-  Error ClientInferStat(InferStat* a) override { return Error::Success; }
+  Error ClientInferStat(InferStat* a) override
+  {
+    a->completed_request_count = local_req_count_;
+    return Error::Success;
+  }
 
  private:
   void LaunchAsyncMockRequest(
-      const InferOptions& options, OnCompleteFn callback)
+      const InferOptions& options_in, OnCompleteFn callback)
```

Comment:
Does it make more sense to change the definition to pass by value (copy) rather than pass a reference and then make a copy of the reference?

Reply:
Yes, it does. Fixed